_posts/2023-08-25-introducing-the-new-kaskada.md

```diff
@@ -0,0 +1,83 @@
+
+## Features for Real-Time AI Applications
+
+Getting started with Generative AI is easier today than it’s ever been:
+
+* Foundation models built by OpenAI, Facebook and others can be used as a starting point, allowing sophisticated applications to be built with a fraction of the data that would otherwise be necessary.
+* Services such as OpenAI eliminate the need to manage complex infrastructure
+* Platforms like HuggingFace have made it easier than ever to share and collaborate on open LLMs.
+
+The New Kaskada compliments these resources, making it easier than ever to utilize real-time data by providing several key components:
+
+### 1. Real-time Aggregation
+
+In a world where data is continuously flowing, being able to efficiently precompute model inputs is invaluable. With Kaskada's real-time aggregation, you can effortlessly:
+
+- Connect with multiple data streams using our robust data connectors.
+- Transform data on-the-go, ensuring that the model receives the most relevant inputs.
+- Perform complex aggregations to derive meaningful insights from streams of data, making sure your AI models always have the most pertinent information.
+- Pause and resume aggregations in the event of process termination.
+
+The result? Faster decision-making, timely insights, and AI models that are always a step ahead.
+
+### 2. Event Detection
+
+Real-time event detection can mean the difference between catching an anomaly and letting it slip through the cracks. The New Kaskada’s event detection system is designed to:
+
+- Expressively describe complex cross-event and cross-entity conditions to use as triggers.
+- Identify important activities and patterns as they occur, ensuring nothing goes unnoticed.
+- Trigger proactive AI behaviors, allowing for immediate actions or notifications based on the detected events.
+
+From spotting fraudulent activities to identifying high-priority user behaviors, Kaskada ensures that important activities are always on your radar.
+
+### 3. History Replay
+
+Past data holds the keys to effective future decisions. With Kaskada's history replay, you can:
+
+- Backtest AI models by revisiting historical data points.
+- Fine-tune models using per-example time travel, ensuring your models are always optimized based on past and present data.
+- Use point-in-time joins to seamlessly merge data from different data sources at a single point in history, unlocking deeper insights and more accurate predictions.
+
+Kaskada ties together the modern real-time AI stack, providing a data foundation for developing and operating AI applications.
+
+## Join the Community
+
+We believe in the transformative power of real-time AI and the possibilities it holds. We believe that real-time data will allow AI to go beyond question-answering to provide proactive, intelligent applications. We want to hear what excites you about real-time AI - Join our Slack community and share your use cases, insights and experiences with the New Kaskada.
```

Comment:
We also want to hear what excites you about "generative AI" (not just real-time AI)